apps/aeutils/test/data/epoch_full.yaml

```diff
@@ -54,6 +54,10 @@ mempool:
     invalid_tx_ttl: 5
     # Mempool (re-)synchronization interval (in ms)
     sync_interval: 1800000
+    # Maximum nonce offset accepted
+    nonce_offset: 5
```

Comment:
Just curious, if this is maximum why the option is not `max_nonce_offset` ?

Reply:
no real reason here... imho `nonce_offset` is precise enough 🤷‍♂️